### ISSUE TITLE
Error: Failed parsing JSON config key prefix

### ISSUE THREAD

**Issue by sebastiandenis** [abehaskins/firepit#9]
Can't start firepit-win:

```
C:\Users\user\Downloads>firepit-win-0-0-7.exe
Please wait while the Firebase CLI downloads...
Error: Failed parsing JSON config key prefix: "C:\Users\user\.cache\firebase\cli"
    at parseField (C:\snapshot\firepit\node_modules\npm\lib\config\core.js:376:13)
    at C:\snapshot\firepit\node_modules\npm\lib\config\core.js:330:24
    at Array.forEach (<anonymous>)
    at Conf.add (C:\snapshot\firepit\node_modules\npm\lib\config\core.js:328:23)
    at ConfigChain.addString (C:\snapshot\firepit\node_modules\npm\node_modules\config-chain\index.js:244:8)
    at Conf.<anonymous> (C:\snapshot\firepit\node_modules\npm\lib\config\core.js:316:10)
    at C:\snapshot\firepit\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:86:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:442:3)
TypeError: Cannot read property 'get' of undefined
    at errorHandler (C:\snapshot\firepit\node_modules\npm\lib\utils\error-handler.js:205:18)
    at C:\snapshot\firepit\node_modules\npm\bin\npm-cli.js:76:20
    at cb (C:\snapshot\firepit\node_modules\npm\lib\npm.js:228:22)
    at C:\snapshot\firepit\node_modules\npm\lib\npm.js:266:24
    at C:\snapshot\firepit\node_modules\npm\lib\config\core.js:83:7
    at Array.forEach (<anonymous>)
    at Conf.<anonymous> (C:\snapshot\firepit\node_modules\npm\lib\config\core.js:82:13)
    at Conf.f (C:\snapshot\firepit\node_modules\npm\node_modules\once\once.js:25:25)
    at Object.onceWrapper (events.js:273:13)
    at Conf.emit (events.js:182:13)
C:\snapshot\firepit\node_modules\npm\lib\utils\error-handler.js:205
  if (npm.config.get('json')) {
                 ^

TypeError: Cannot read property 'get' of undefined
    at process.errorHandler (C:\snapshot\firepit\node_modules\npm\lib\utils\error-handler.js:205:18)
    at process.emit (events.js:182:13)
    at process._fatalException (internal/bootstrap/node.js:484:27)
```

**Comment by abeisgoat** [comment]
Thanks for filing this!

* What happens if you double-click the .exe from Explorer? Does it just crash?
* Can you check if `C:\Users\user\.cache\firebase\cli` exists?
* Can you check if `C:\Users\user\.cache\firebase\bin\npmrc` exists? If so, what's in it?
* Is `user` your actually account or is that masked? If it's not the actual account name, does the actual one contain non-ASCII characters or spaces?

Thanks a ton!
abe

**Comment by DenisVCode** [comment]
+1

1) cmd appears briefly so you can hardly see the error then closes
2) it does not
3) prefix = "C:\Users\denis\.cache\firebase\cli"
4) no non-ASCII characters or spaces

**Comment by sebastiandenis** [comment]
The same for me.
1. The same message as with command line, but with double-click it closes the window after the error and it is hard to read anything.
2. `C:\Users\user\.cache\firebase\cli` doesn't exist.
3) `C:\Users\user\.cache\firebase\bin\npmrc` exists with `prefix = "C:\Users\user\.cache\firebase\cli"`.
4) I've tried with different user's accounts and using different Windows machines but the result is always the same, i.e. `prefix = "C:\Users\Sebastian\.cache\firebase\cli"`

**Comment by abeisgoat** [comment]
Alright, I can repro this. I'll be taking a look today! Thanks for all the info!

**Comment by abeisgoat** [comment]
Should be fixed, try `0.0.8` and let me know if it's still broken!
